Source/Core/Core/PowerPC/JitCommon/JitCache.cpp

```diff
@@ -132,12 +132,18 @@ void JitBaseBlockCache::FinalizeBlock(JitBlock& block, bool block_link,
     LinkBlock(block);
   }
 
-  if (Symbol* symbol = g_symbolDB.GetSymbolFromAddr(block.effectiveAddress))
+  Symbol* symbol = nullptr;
+  if (!SConfig::GetInstance().m_perfDir.empty() &&
```

Comment:
May you create a small function within JitRegister which checks if it will do anything at all? IIRC there is also a vtune implementation which doesn't check this variable.